src/index.html

```diff
@@ -6,8 +6,361 @@
+                                      <p class="level3">Proficient</p>
+                                    </div>
+                                    <div class="skill">
+                                      <p class="sname3">PowerPoint</p>
+                                      <p class="level4">Proficient</p>
+                                    </div>
+                                    <div class="skill">
+                                      <p class="sname4">French</p>
+                                      <p class="level5">Native</p>
+                                    </div>
+                                    <div class="skill">
+                                      <p class="sname5">Spanish</p>
+                                      <p class="level6">Advanced</p>
+                                    </div>
+                                  </div>
+
+                                    <!-- QUICKID PART-->
+                                  <div class="quickid">
+                                    <div class="quickid_section">
+                                      <div class="quickid_title">
+                                        <h2>QUICK ID</h2>
+                                      </div>
+                                      <table>
+                                        <tr>
+                                          <td>Phone</td>
+                                          <td>123-456-7890</td>
+                                        </tr>
+
+                                        <tr>
+                                          <td>Email</td>
+                                          <td><a href="mailto:info@mysite.com">info@mysite.com</a></td>
+                                        </tr>
+
+                                        <tr>
+                                          <td>Website</td>
+                                          <td><a href="">www.wix.com</a></td>
+                                        </tr>
+
+                                        <tr>
+                                          <td>Address</td>
+                                          <td>500 Terry Francois Street, San Francisco, CA 94158</td>
+                                        </tr>
+
+                                        
+
+                                      </table>
+                                    </div>
+                                  </div>
+
+                                  <div class="contact">
+                                    <div class="con_title">
+                                      <h2>CONTACT ME</h2>
+                                    </div>
+                                    <div class="contactbox"></div>
+                                    <div class="con_content">
+                                      <form action="/action_page.php" method="post">
+                                        <input type="text" placeholder="Name" />
```

Comment:
form input without label is considered to be bad for usability and accessibility, we will go over it on lesson 2.